addons/mail/static/src/components/chatter_container/chatter_container.js

```diff
@@ -17,6 +11,8 @@ const components = { Chatter };
  * may attempt to create a chatter before messaging has been initialized, so
  * this component delays the mounting of chatter until it becomes initialized.
  */
+const { Component } = owl;
+
```

Comment:
why diff on this line? keep it where it was

Reply:
Lots of search-replace during this, just missed this one when reviewing the diff.